docs/terminal/sydney.md

```diff
@@ -340,63 +349,72 @@ Any aircraft that don't meet these criteria must be coordinated to SY TCU with a
-### SY TCU / BK TWR
+### SY TCU / BK ADC
 #### Departures
 
-Aircraft departing YSBK in to SY TCU Class C will be coordinated from **BK TWR** at Taxi. Aircraft will need to be passed airways clearances to **BK TWR** at this point, to be relayed to the aircraft.
+Aircraft departing YSBK in to SY TCU Class C will be coordinated from **BK ADC** at Taxi. Aircraft will need to be passed airways clearances to **BK ADC** at this point, to be relayed to the aircraft.
 
 !!! example
-    **BK TWR** -> **SY TCU**: "Taxi, TFX12 for YMML via WOL"  
-    **SY TCU** -> **BK TWR**: "TFX12, BK, WOL, Flight Planned Route, A030, Squawk 3601"  
-    **BK TWR** -> **SY TCU**: "BK, WOL, Flight Planned Route, A030, Squawk 3601, TFX12"  
-    **BK TWR** Will then pass the airways clearance to TFX12
+    **BK ADC** -> **SY TCU**: "Taxi, TFX12 for YMML via WOL"  
+    **SY TCU** -> **BK ADC**: "TFX12, BK, WOL, Flight Planned Route, A030, Squawk 3601"  
+    **BK ADC** -> **SY TCU**: "BK, WOL, Flight Planned Route, A030, Squawk 3601, TFX12"  
 
-**BK TWR** will then give a "Next" call, where the SY TCU controller shall assign a heading (usually 290, for separation from YSSY).
+    **BK ADC** Will then pass the airways clearance to TFX12  
+    `AIP GEN 3.4`
+
+**BK ADC** will then give a "Next" call, where the SY TCU controller shall assign a heading (usually 290, for separation from YSSY).
 
 !!! example
-    **BK TWR** -> **SY TCU**: "Next, TFX12"  
-    **SY TCU** -> **BK TWR**: "TFX12, Right Heading 290, A030"  
-    **BK TWR** -> **SY TCU**: "Right Heading 290, A030, TFX12"  
-    **BK TWR** Will then clear the aircraft to takeoff with the assigned heading, and instruct them to contact SY TCU passing A020.
+    **BK ADC** -> **SY TCU**: "Next, TFX12"  
+    **SY TCU** -> **BK ADC**: "TFX12, Right Heading 290, A030"  
+    **BK ADC** -> **SY TCU**: "Right Heading 290, A030, TFX12"  
+
+    **BK ADC** Will then clear the aircraft to takeoff with the assigned heading, and instruct them to contact SY TCU passing A020.  
+    `AIP GEN 3.4`
 
 #### Arrivals
 
-YSBK arrivals shall be coordinated to **BK TWR** from SY TCU prior to transfer of jurisdiction.
+YSBK arrivals shall be coordinated to **BK ADC** from SY TCU prior to transfer of jurisdiction.
 
 !!! tip
-    Ensure the aircraft's FDR is up-to-date in order to give **BK TWR** maximum situational awareness of the traffic picture. (eg. if the aircraft is doing the RNP approach, ensure the FDR has been rerouted via the appropriate points)
+    Ensure the aircraft's FDR is up-to-date in order to give **BK ADC** maximum situational awareness of the traffic picture. (eg. if the aircraft is doing the RNP approach, ensure the FDR has been rerouted via the appropriate points)
 
 !!! example
-    **SY TCU** -> **BK TWR**: "ABC, via ODALE, Number 1"  
-    **BK TWR** -> **SY TCU**: "ABC, Number 1"  
+    **SY TCU** -> **BK ADC**: "ABC, via ODALE, Number 1"  
+    **BK ADC** -> **SY TCU**: "ABC, Number 1"   
```

Comment:
Remove sequence number

Reply:
and change to heads-up format. ie "via ODALE, ABC", rather than "ABC, via ODALE"